src/text-field/README.md

```diff
@@ -2,7 +2,33 @@
 
 The `vscode-text-field` is a web component implementation of a [text field element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/text).
 
-## Attributes
+![Text field hero](/docs/assets/text-field-hero.png)
+
+## Usage
+
+| ❌ Don't                                                                                                                                    | ✅ Do                                                  |
+| ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
+| ![Image placeholder](/docs/assets/img-placeholder.png)                                                                                      | ![Image placeholder](/docs/assets/img-placeholder.png) |
+| Don't use a text field for inputs that are greater than a single line of text. Use a [text-area](../text-area/README.md) component instead. | Use text fields for short input values.                |
+
+| ❌ Don't                                                                      | ✅ Do                                                                     |
+| ----------------------------------------------------------------------------- | ------------------------------------------------------------------------- |
+| ![Image placeholder](/docs/assets/img-placeholder.png)                        | ![Image placeholder](/docs/assets/img-placeholder.png)                    |
+| Don't use a placeholder value instead of a label unless absolutely necessary. | Use descriptive labels to help users understand the text field's purpose. |
+
+| ❌ Don't                                                   | ✅ Do                                                             |
+| ---------------------------------------------------------- | ----------------------------------------------------------------- |
+| ![Image placeholder](/docs/assets/img-placeholder.png)     | ![Image placeholder](/docs/assets/img-placeholder.png)            |
+| Don't include critical information in a placeholder value. | Use additional captions if necessary to provide more information. |
```

Comment:
I assume by "captions" you probably mean "placeholders". If so could you switch to using "placeholders"?

Reply:
I should actually change this to say "helper text" or similar. The idea here is to ensure placeholder text isn't used for details that shouldn't be hidden away once there is input. I'll try phrasing this differently